tests/NHSD.BuyingCatalogue.Organisations.Api.UnitTests/Builders/ServiceRecipientBuilder.cs

```diff
@@ -0,0 +1,38 @@
+﻿using NHSD.BuyingCatalogue.Organisations.Api.Models;
+
+namespace NHSD.BuyingCatalogue.Organisations.Api.UnitTests.Builders
+{
+    internal sealed class ServiceRecipientBuilder
+    {
+        private string _name;
+        private string _odsCode;
+
+        private ServiceRecipientBuilder(int index)
+        {
+            _name = $"Service {index}";
```

Comment:
Didnt we decide in the code standards meeting it would just be an empty constructor? And not initialise anything in there

Reply:
Nah we decided not to construct the actual ServiceRecipient here, but setting up defaults values for parameters should be fine